ci/scripts/python_sdist_build.sh

```diff
@@ -22,6 +22,8 @@ set -ex
 source_dir=${1}
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
+git config --global --add safe.directory ${source_dir}
```

Comment:
ah, most of the other jobs stuff this into the docker-compose.yml command line, since I don't want these to mutate global state if run outside of docker

Reply:
ah i see, i'll fix that :)